.config

```diff
@@ -0,0 +1,906 @@
+#
```

Comment:
@3mdeb-karolzmyslowski .config should never be commited. How did you do that anyway? It should be in the `.gitignore`.

Reply:
(Corrected, in `https://github.com/Dasharo/coreboot/pull/309/commits/a5dc727a9bd297e534f3e646375a4c5e2ebe70b4`)